module_2

```diff
@@ -0,0 +1,14 @@
+odd_list = []
+def odd_return(n):
+    for num in range(n):
+       if num % 2 == 0:
+           odd_list.append(num+1)
```

Comment:
Instead of appending the values to a global list, create a list inside the function, append to that, and then return the list using the `return` command.